src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,35 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        List<String> result = new ArrayList<>();
+
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fileName));
+            StringBuilder stringBuilder = new StringBuilder();
+            String value = reader.readLine();
+            while (value != null) {
+
+                for (String word : value.split(" ")) {
```

Comment:
```suggestion
                for (String word : value.toLowerCase().split("\\W+")) {
```
this way you you won't need to cast each word to lowercase and call replaceAll